wowup-electron/src/assets/i18n/de.json

```diff
@@ -217,19 +217,19 @@
         "GAME_VERSION_COLUMN_HEADER": "Spielversion",
         "LATEST_VERSION_COLUMN_HEADER": "Aktuelle Version",
         "PROVIDER_COLUMN_HEADER": "Anbieter",
-        "PROVIDER_RELEASE_CHANNEL": "Anbieter Kanal",
-        "RELEASED_AT_COLUMN_HEADER": "Veröffentlicht am",
+        "PROVIDER_RELEASE_CHANNEL": "Anbieter-Channel",
+        "RELEASED_AT_COLUMN_HEADER": "Erscheinungsdatum",
         "STATUS_COLUMN_HEADER": "Status",
         "UPDATED_AT_COLUMN_HEADER": "Aktualisiert vor"
       },
       "UNINSTALL_POPUP": {
-        "CONFIRMATION_ACTION_EXPLANATION": "Die Entfernung eines Addons über WowUp wird das ausgewählte Addon auch von deinem 'interfaces/addons' Ordner entfernen. Die Charakter Einstellungen des Addons werden nicht entfernt.",
-        "CONFIRMATION_LESS_THAN_THREE": "Bist Du sicher, dass Du die folgenden {count} Addons entfernen möchtest?",
-        "CONFIRMATION_MORE_THAN_THREE": "Bist Du sicher, dass Du die ausgewählten {count} Addons entfernen möchtest?",
-        "CONFIRMATION_ONE": "Bist Du sicher, dass Du {addonName} entfernen möchtest?",
-        "DEPENDENCY_MESSAGE": "{addonName} hat {dependencyCount} {dependencyCount, plural, one{Abhängigkeit} other{Abhängigkeiten}}. Möchtest Du diese ebenfalls entfernen?",
-        "DEPENDENCY_TITLE": "Addon Abhängigkeiten entfernen?",
-        "TITLE": "{count, plural, =1{Addon} other{Addons}} deinstallieren?"
+        "CONFIRMATION_ACTION_EXPLANATION": "Dadurch werden alle zugehörigen Ordner aus dem World of Warcraft-Ordner entfernt.",
```

Comment:
Das ist nicht richtig, der WTF Order wird nicht angerührt.
"Dadurch werden alle zugehörigen Ordner aus dem World of Warcraft Interface-Ordner entfernt. Die Charakter Einstellungen des Addons werden nicht entfernt."

Reply:
Wieder good catch, hatte einfach das korrigiert was davor stand und nicht ganz darüber nachgedacht. Mein Fehler!